src/CommunityToolkit.Maui.Core/Capabilities/PlatformVersion/PlatformVersion.android.cs

```diff
@@ -0,0 +1,16 @@
+﻿using Android.OS;
+
+namespace CommunityToolkit.Maui.Core.Capabilities;
+
+/// <summary>
+/// Helper class that stores the platform version.
+/// </summary>
+public static class PlatformVersion
+{
+	static int? sdkInt;
+
+	/// <summary>
+	/// Android version running on the device
+	/// </summary>
+	public static int SdkInt => sdkInt ??= (int)Build.VERSION.SdkInt;
```

Comment:
Cannot `DeviceInfo.Version` be used for this? This creates another source of truth for getting device version

Reply:
@pictos Why is this resolved? I don't see any related commit. Could you at least comment on why this code is nesesary? (It was needed in XCT since Essentials were in the separate NuGet)

Reply:
This code is more performatic than Maui ones. The core project is meant to be used by other toolkit implementors and not by app projects, so devs should not use it.
I added one more line in the `summary` telling this code is for internal use only.